README.md

```diff
@@ -33,6 +33,8 @@ In general, you call the cli like so: <br>
 ### CLI options
 | Option                      	| Shortcut 	| Description                                                                           	| Default 	|
 |-----------------------------	|----------	|---------------------------------------------------------------------------------------	|---------	|
+| --benchmark                  	| -b n       	| Execute compiler benchmarks. <n> is the number of benchmark runs	                	| 0      	|
```

Comment:
there's still the `<n>`

Reply:
and maybe you can print these parameters in italics in the Readme?

Reply:
Oh, forgot that occurence. Is fixed now